src/main/java/core/basesyntax/service/DataProcessor.java

```diff
@@ -1,25 +1,29 @@
-package core.basesyntax.dataservice;
+package core.basesyntax.service;
 
 import core.basesyntax.db.Storage;
 import core.basesyntax.exceptions.OperationNotFoundException;
-import core.basesyntax.operationstrategy.Operation;
+import core.basesyntax.model.Operation;
 import core.basesyntax.operationstrategy.OperationStrategy;
-import core.basesyntax.operationstrategy.OperationStrategyImpl;
 import java.util.Map;
 import java.util.Set;
 
-public class ProcessData {
+public class DataProcessor {
     private static final String LINE_SEPARATOR = System.lineSeparator();
     private static final String COMA_SPLITTER = ",";
     private static final String HEAD_OF_THE_OUTPUT_FILE = "fruit,quantity" + LINE_SEPARATOR;
-    private static final int indexOfOperationsType = 0;
-    private static final int indexOfFruitType = 1;
-    private static final int indexOfQuantity = 2;
-    private final OperationStrategy operationStrategy = new OperationStrategyImpl();
+    private static final int INDEX_OF_OPERATIONS_TYPE = 0;
+    private static final int INDEX_OF_FRUIT_TYPE = 1;
+    private static final int INDEX_OF_QUANTITY = 2;
+    private static final int INDEX_OF_BEGINNING_OF_DATA = 1;
+    private final OperationStrategy operationStrategy;
+
+    public DataProcessor(OperationStrategy operationStrategy) {
+        this.operationStrategy = operationStrategy;
+    }
 
     public String processData(String dataFromFile) {
         String[] data = dataFromFile.split(LINE_SEPARATOR);
-        for (int i = 1; i < data.length; i++) {
+        for (int i = INDEX_OF_BEGINNING_OF_DATA; i < data.length; i++) {
```

Comment:
I meant something like FRUIT_NAME_INDEX its make clearly from which index you start iterating